### DISCUSSION TITLE
Whatsapp-Bot Python kirimwa.id

### DISCUSSION THREAD

**Discussion by rzpr** [kirimwa-id/forum#47] [Pamerkan]
Halo, Saya Mau Sharing Demo Project Whatsapp Bot Saya Versi Python  Bisa Lihat Di :

 https://github.com/rezzaapr/kirimwaid-whatsapp-bot

**Comment by rioastamal** [comment]
Terima kasih mas @rezzaapr sudah posting penggunaan API KirimWA.id disini. Ditunggu update karya berikutnya 👍
